build/NuGetPackages/Microsoft.Build.Runtime.nuspec

```diff
@@ -33,61 +33,62 @@
+    <file src="$outputPath$Microsoft.CSharp.CrossTargeting.targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.CSharp.CurrentVersion.targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.CSharp.targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.Data.Entity.targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.NETFramework.CurrentVersion.props" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.NETFramework.CurrentVersion.targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.NETFramework.props" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.NETFramework.targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.ServiceModel.targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.VisualBasic.CrossTargeting.targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.VisualBasic.CurrentVersion.targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.VisualBasic.targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.VisualStudioVersion.v11.Common.props" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.VisualStudioVersion.v12.Common.props" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.VisualStudioVersion.v14.Common.props" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.WinFx.targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.WorkflowBuildExtensions.targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.Xaml.targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Workflow.Targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Workflow.VisualBasic.Targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
 
     <!--
       contentFiles\any\netcoreapp1.0
     -->
-    <file src="$outputPathNetCore$MSBuild.dll" target="contentFiles\any\netcoreapp1.0\" buildAction="None" copyToOutput="true"/>/>
```

Comment:
I thought the same thing. Turns out by default it gets escaped and is completely ignored!

```C#
var doc = new XmlDocument();
doc.Load(new StringReader(@"<files> <file /> /> </files>"));
void print(XmlLinkedNode element) => Console.WriteLine($"{element.NodeType}: {element.Name} = {element.Value}");
doc.ChildNodes.Cast<XmlLinkedNode>().Traverse(element => element.ChildNodes.Cast<XmlLinkedNode>()).ForEach(print);
```
=>
```
Element: files =
Element: file =
Text: #text =  />
```
You get a nice `XmlText` element mixed in there and if you don't check for it you'll never know was there...